examples/galton.jl

```diff
@@ -49,31 +49,32 @@ function draw_pins(n, xpos, ypos, offset; color = "white")
 end
 
 """
-    flip_coin()
+    flip_coin(p)
 
-Simulates flipping a coin, returning -1 and 1 with equal probability. Helper function used in `move_ball`.
+Simulates flipping a coin, returning -1 with probability `p` and 1 with probability (1 - p). Helper function
+used in `move_ball`.
 """
-function flip_coin()
-    return rand() < 0.5 ? 1 : -1
+function flip_coin(p)
+    return rand() < p ? 1 : -1
 end
 
 """
-    move_ball(ball, first, last, offset, fno)
+    move_ball(ball, first, last, offset, p, fno)
 
 Moves `ball` from the `first` level of pins to the `last` level of pins (i.e. top to bottom). `offset` should
-be the same offset used when creating the bins using `draw_separators`. The value `fno` is the frame number
-from which the ball should start moving.
+be the same offset used when creating the bins using `draw_separators`. The value `p` is the probability that
+a ball will go left at each pin, and the value `fno` is the frame number from which the ball should start moving.
```

Comment:
I think it's the probability that it will go right (positive x direction), isn't it?